### ISSUE TITLE
Update lit-element / granite-lit-bulma to last version

### ISSUE THREAD

**Issue by GildasMorel** [sfeir-open-source/talk-control#80]
- Checkout the develop branch
- Update lit-element and granite-lit-bulma dependencies to last version.
- Run `npm i` and `npm start` and verify that there are no console errors.

**Comment by karthikshetty03** [comment]
Can I work on this issue ?

**Comment by GildasMorel** [comment]
Of course, I assigned the issue to you, tell me if you need more informations.
I hope the description is clear enough.

Thanks for your help.

**Comment by karthikshetty03** [comment]
@GildasMorel what is the last version are you referring to ?

**Comment by GildasMorel** [comment]
I mean the last version of granite-lit-bulma (0.9.0-4) on 
https://www.npmjs.com/package/@granite-elements/granite-lit-bulma.

I tried to put the version 0.9.0-4 in package.json and to update lit-element version to the last to, but I had some javascript errors at launch.
I didn't go further at the time.

**Comment by karthikshetty03** [comment]
```
⚠️  Could not load existing sourcemap of "../../../node_modules/@granite-elements/granite-lit-bulma/granite-lit-bulma.js".
⚠️  Could not load existing sourcemap of "../../../node_modules/@granite-elements/granite-lit-bulma/granite-lit-bulma.js".

```
I am getting this on running npm start for the very first time after updating the versions and doing npm i
Stopping the server and doingnpm start again does not give any error messsage on console
Also build is successful in both cases
Should I look for a fix for the first case ?

**Comment by karthikshetty03** [comment]
![Screenshot from 2020-10-01 01-41-04](https://user-images.githubusercontent.com/45932358/94734716-40596200-0387-11eb-824d-0a06f0182ab0.png)
I am getting the two warning errors on running `npm start` only for the first time after running `npm i`
Do they need to be fixed ?

**Comment by GildasMorel** [comment]
No, actually these two messages are just warnings.
The problem is when you open http://localhost:3000 in your browser and you get this :
![tc-bulma-error](https://user-images.githubusercontent.com/50823695/94741736-ddf76600-0374-11eb-9be5-589186f1043c.png)

**Comment by GildasMorel** [comment]
Here is what I did :
- update @granite-elements/granite-lit-bulma version to ^0.9.0-4
- npm i
- npm start
- open my browser on localhost:3000.

This shows a blank page with the heading "TalkControl".
And you will see the error above.
That's what I would like to be fixed.

**Comment by karthikshetty03** [comment]
There was a problem with the import statement
I have fixed it now

**Comment by karthikshetty03** [comment]
@GildasMorel Changing the import statements fixed these issues
I have created a PR #87

**Comment by GildasMorel** [comment]
Thanks a lot, I will review your PR by the end of the day.
I'll go back to you when it's done.

**Comment by GildasMorel** [comment]
Your PR has been merged, thanks a lot for your help.

**Comment by jefBinomed** [comment]
Fermeture (audit v0.5.0) : lit-element est déjà à la dernière majeure (^4.2.2) suite à la migration Node 24 / TypeScript strict.
